app/controllers/reputation_controller.rb

```diff
@@ -25,6 +25,18 @@ def reply
     register_thumbup_reply_activity if params[:up_down] == 'up'
   end
 
+  def advertisement
+    if user_signed_in?
+      @advertisement = Advertisement.find(params[:id])
+      @evaluation_value = params[:up_down] == 'down' ? 0 : 1
+      @advertisement.add_or_update_evaluation(:likes, @evaluation_value, current_user)
+    else
+      # TODO: ja.ymlを使うようにする
+      @error_message = 'ログインが必要です'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.